### ISSUE TITLE
New portal should have scratch place

### ISSUE THREAD

**Issue by isimluk** [isimluk/openscap-portal#4]
+ Hosting new guides
+ Hosting screenshots needed by openscap base docuementation
+ Hosting our API documentation

**Comment by mpreisler** [comment]
I have created http://static-rhopenscap.rhcloud.com as a Silver 6GB OpenShift app. It will be accessible via http://static.open-scap.org in the future. The DNS change request was filed but hasn't been completed yet.

In the meantime I will upload openscap doxygen docs, SCAP Workbench manual and SSG guides there. We have discussed in the office that we only want to host docs for each maintenance branch, not for each version. The versions in the same branch are similar enough to do that.

**Comment by mpreisler** [comment]
I have uploaded the basics, check http://static-rhopenscap.rhcloud.com/

I am trying to figure out how to make the public jenkins instance do all the work in the future :-)

**Comment by mpreisler** [comment]
A job has been created that builds all the docs and guides, see https://jenkins.open-scap.org/job/static_openscap_docs

Right now it's blocked on `openscap-devel`, `zip` and `asciidoctor` not being available on any of the slaves. I guess we will build this on the planned Fedora slave only.

**Comment by mpreisler** [comment]
The Jenkins job works and builds everything needed for the `static.open-scap.org` page. The DNS change has been completed.

http://static.open-scap.org/

The landing page needs some visual tweaks but other than that I consider this done. The visual tweaks can be done in another ticket.
